src/main/java/seedu/address/logic/commands/FilterPaidCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.StudentHasPaidPredicate;
+
+/**
+ * Finds and lists all persons in address book whose paid status matches the argument.
+ */
+public class FilterPaidCommand extends Command {
+    public static final String COMMAND_WORD = "filterp";
```

Comment:
Maybe the command word could be changed to something more intuitive for the user? Maybe something like "filterpaid"